[FW][IMP] Add secondary currency into revaluation view

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/stock#675

maq-adhoc committed:
[IMP] Add secondary currency into revaluation view

X-original-commit: 50b58dfa5384e176f4b47ff4e1d3580a6b94ba30

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/stock/pull/676).

fw-bot-adhoc commented:
@maq-adhoc @rov-adhoc cherrypicking of pull request ingadhoc/stock#675 failed.

stdout:
```
CONFLICT (modify/delete): stock_currency_valuation/wizard/stock_valuation_layer_revaluation.py deleted in HEAD and modified in a7fad399 ([IMP] Add secondary currency into revaluation view). Version a7fad399 ([IMP] Add secondary currency into revaluation view) of stock_currency_valuation/wizard/stock_valuation_layer_revaluation.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/models/stock_valuation_layer.py deleted in HEAD and modified in a7fad399 ([IMP] Add secondary currency into revaluation view). Version a7fad399 ([IMP] Add secondary currency into revaluation view) of stock_currency_valuation/models/stock_valuation_layer.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/models/account_move.py deleted in HEAD and modified in a7fad399 ([IMP] Add secondary currency into revaluation view). Version a7fad399 ([IMP] Add secondary currency into revaluation view) of stock_currency_valuation/models/account_move.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/__manifest__.py deleted in HEAD and modified in a7fad399 ([IMP] Add secondary currency into revaluation view). Version a7fad399 ([IMP] Add secondary currency into revaluation view) of stock_currency_valuation/__manifest__.py left in tree.

```

stderr:
```
19:03:57.831200 git.c:444               trace: built-in: git cherry-pick a7fad39968e0f0c19681c8e5bd6b765729acf8dc
error: could not apply a7fad399... [IMP] Add secondary currency into revaluation view
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

maq-adhoc commented:
Ciero va derecha a 18
